Automattic/pocket-casts-ios
# Playback 2024: Intro story

| 📘 Part of: #2250 |
|:---:|
| **Depends on #2306** |

Completes https://github.com/Automattic/pocket-casts-ios/issues/2314

Adds an Intro screen as well as a "Top Spot" screen just as a placeholder for now

https://github.com/user-attachments/assets/972260bd-de38-4725-b1ea-69bfba1133b7

### Code Changes

 * Humane Medium font was added
 * Several sticker SVGs were added
 * Added a `PositionModified` to be able to specify a position with a given corner to avoid writing out code to reposition the object based on its corner (the values that are easiest to get from Figma).
 * Added suffix to the previous story types to avoid confusion with 2024

## To test

* Enable the `endOfYear2024` feature flag
* Restart the app
* Tap the "Playback 2024 is here!" cell
* Verify that the intro screen appears with animation

## Checklist
